libloki/crypto.js

```diff
@@ -329,10 +329,13 @@
    * This handles specific session reset logic that we need.
    */
   class LokiSessionCipher {
-    constructor(storage, address) {
+    constructor(storage, protocolAddress) {
       this.storage = storage;
-      this.address = address;
-      this.sessionCipher = new libsignal.SessionCipher(storage, address);
+      this.protocolAddress = protocolAddress;
```

Comment:
Aren't we mocking `libsignal.SessionCipher`, what if something expects this.address ?

Reply:
No we're not. `LokiSessionCipher` is only specifically meant to be used in `decryption` only. We don't inherit or forward any other `SessionCipher` functions. Like the description states it's only a wrapper around `SessionCipher.decrypt`